scripts/dev/bless_tests.php

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env php
+<?php
+
+if ($argc < 2) {
+    die("Usage: php bless_tests.php dir/");
+}
+
+$dir = $argv[1];
+$it = new RecursiveIteratorIterator(
+    new RecursiveDirectoryIterator($dir),
+    RecursiveIteratorIterator::LEAVES_ONLY
+);
+foreach ($it as $file) {
+    $path = $file->getPathName();
+    if (!preg_match('/^(.*)\.phpt$/', $path, $matches)) {
+        // Not a phpt test
+        continue;
+    }
+
+    $outPath = $matches[1] . '.out';
+    if (!file_exists($outPath)) {
+        // Test did not fail
+        continue;
+    }
+
+    $phpt = file_get_contents($path);
+    if (false !== strpos($phpt, '--XFAIL--')) {
+        // Don't modify expected output of XFAIL tests
+        continue;
+    }
+
+    $out = file_get_contents($outPath);
+    $out = normalizeOutput($out);
+    $phpt = insertOutput($phpt, $out);
+    file_put_contents($path, $phpt);
+}
+
+function normalizeOutput(string $out): string {
```

Comment:
Maybe we can expand this function to receive something from the command line. This script would help me a lot in #3429

Reply:
Not sure I understand what you mean here. Could you give an example?

Reply:
The function current has hard-coded to only fix "Resource id". I wonder if we can have some way to provide what is gonna to be changed, as #3429. Just an idea, nothing that blocks its merge as it is.